RadarSDK/Radar.m

```diff
@@ -244,101 +244,15 @@ + (void)trackOnceWithLocation:(CLLocation *)location completionHandler:(RadarTra
+    [[RadarVerificationManager sharedInstance] trackVerifiedTokenWithCompletionHandler:completionHandler];
+}
 
-                                     [[RadarVerificationManager sharedInstance]
-                                         getAttestationWithNonce:config.nonce
-                                               completionHandler:^(NSString *_Nullable attestationString, NSString *_Nullable keyId, NSString *_Nullable attestationError) {
-                                                   [[RadarAPIClient sharedInstance]
-                                                       trackWithLocation:location
-                                                                 stopped:RadarState.stopped
-                                                              foreground:[RadarUtils foreground]
-                                                                  source:RadarLocationSourceForegroundLocation
-                                                                replayed:NO
-                                                                 beacons:nil
-                                                                verified:YES
-                                                       attestationString:attestationString
-                                                                    keyId:keyId
-                                                        attestationError:attestationError
-                                                               encrypted:YES
-                                                       completionHandler:^(RadarStatus status, NSDictionary *_Nullable res, NSArray<RadarEvent *> *_Nullable events,
-                                                                           RadarUser *_Nullable user, NSArray<RadarGeofence *> *_Nullable nearbyGeofences,
-                                                                           RadarConfig *_Nullable config, NSString *_Nullable token) {
-                                                            if (status == RadarStatusSuccess && config != nil) { 
-                                                                [[RadarLocationManager sharedInstance] updateTrackingFromMeta:config.meta];
-                                                            }
-                                                           if (completionHandler) {
-                                                               [RadarUtils runOnMainThread:^{
-                                                                   completionHandler(status, token);
-                                                               }];
-                                                           }
-                                                       }];
-                                               }];
-                                 }];
-            }];
++ (void)startTrackingVerified:(BOOL)token {
+    [[RadarVerificationManager sharedInstance] startTrackingVerified:token];
```

Comment:
as this function potentially takes a whole Interval (presently 300s) to call `trackVerified`, do we want to manually call `trackVerified` and then start the timer?

Reply:
Good call. In practice it seems to get called right away because the "network connected" callback seems to fire right away, but I added an explicit call

I also added some synchronization with `@synchronized(self)`

Reply:
I ended up removing the synchronization bc I don't think it's needed with the logic in `RadarAPIHelper`. Can always add it back later if needed